src/main/java/com/github/cameltooling/lsp/internal/parser/CamelKafkaConnectDSLParser.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cameltooling.lsp.internal.parser;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.Properties;
+
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.TextDocumentItem;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.cameltooling.lsp.internal.instancemodel.CamelURIInstance;
+import com.github.cameltooling.lsp.internal.instancemodel.PropertiesDSLModelHelper;
+
+/**
+ * @author Aurelien Pupier
+ * Restriction on notation for properties file, it must be with '=' and without spaces
+ */
+public class CamelKafkaConnectDSLParser extends ParserFileHelper {
+	
+	public static final String CAMEL_SINK_URL = "camel.sink.url";
+	public static final String CAMEL_SOURCE_URL = "camel.source.url";
+	private static final Logger LOGGER = LoggerFactory.getLogger(CamelKafkaConnectDSLParser.class);
+
+	@Override
+	public String getCamelComponentUri(String line, int characterPosition) {
+		if (line.startsWith(CAMEL_SOURCE_URL) && CAMEL_SOURCE_URL.length() < characterPosition 
+				|| line.startsWith(CAMEL_SINK_URL) && CAMEL_SINK_URL.length() < characterPosition) {
```

Comment:
indentation problem?

Reply:
seems to be the normal behavior with Eclipse defaults codestyle

I think it is displayed quite well with default size of tabs:
![image](https://user-images.githubusercontent.com/1105127/62113594-2b7dcc00-b2b5-11e9-872a-6f566c321da9.png)